tasks/installation.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: install groups related packages
+  package:
+    name: '{{ groups_packages }}'
+    state: present
+  register: groups_register_packages_installed
+  until: groups_register_packages_installed is succeeded
+  tags:
```

Comment:
We don't add tags on task level, only in the main.yml. What's up with that `until`?

Reply:
Took this from ansible-role-users. Needs to be update there too, imho.

Until: See https://github.com/adfinis-sygroup/ansible-role-icinga2_web/pull/1/commits/b5e8983a1a363476bf1e6d008cc5eadf38bebaa4 same issue.

Reply:
fixed as suggested by @fujexo